bytecomp/bytepackager.ml

```diff
@@ -233,9 +252,13 @@ let package_object_files ~ppf_dump files targetfile targetname coercion =
         | { pm_kind = PM_intf } ->
             required_globals
         | { pm_kind = PM_impl { cu_required_globals; cu_reloc } } ->
+            let cu_required_globals =
+              List.map Ident.create_persistent cu_required_globals
+            in
```

Comment:
Maybe we should change the type of `cu_required_globals` to use globals instead of identifiers?

Reply:
I've a commit doing that in https://github.com/hhugo/ocaml/tree/represent-ids-with-standard-types-in-cmo-files.
It removes a bunch of `Ident.create_persistent`